VideoLocker/src/main/java/org/edx/mobile/model/api/CourseEntry.java

```diff
@@ -248,22 +252,72 @@ public boolean isGroupAvailable(SocialUtils.SocialType type) {
 
     }
 
-    public String getDescription(Context context){
+    @SuppressLint("SimpleDateFormat")
+    public String getDescription(Context context) {
         StringBuilder detailBuilder = new StringBuilder();
-        if ( getOrg() != null){
-            detailBuilder.append( getOrg());
+
+        if (getOrg() != null) {
+            detailBuilder.append(getOrg());
```

Comment:
I think we can use the local variables directly here instead of using the getter methods :wink: